core/src/test/java/org/elasticsearch/index/mapper/DynamicMappingDisabledIT.java

```diff
@@ -0,0 +1,29 @@
+package org.elasticsearch.index.mapper;
+
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.test.ESIntegTestCase;
+
+import java.io.IOException;
+
+/**
+ * Created by davidgalbraith on 12/17/15.
+ */
+public class DynamicMappingDisabledIT extends ESIntegTestCase {
```

Comment:
Is there any way to just make this a unit test (extending `ESTestCase`) instead of a full blown integration test?